File: modulo 1/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int tamanho, i = 0, h = 0, vazio = 0;
    do
    {
        tamanho = get_int("Tamanho de 1 a 8: \n"); //pede tamanho
    }
    while (tamanho <= 0 || tamanho > 8);

    for (h = 1; h <= tamanho; h++) //imprime a piramide
    {

        for (vazio = tamanho - h; vazio >= 1; vazio--) //adiciona espaço
        {
            printf(" ");
        }

        for (i = 1; i <= h; i++) //adiciona caracteres para largura
        {
            printf("#");
        }
        printf("  "); //espaço no meio

        for (i = 1; i <= h; i++) //adiciona caracteres da direita
        {
            printf("#");
        }


        printf("\n");
    }


}

File: modulo 6/mario.py
from cs50 import get_int



def main():
    tamanho = 0
    while tamanho > 8 or tamanho <= 0:
        tamanho = get_int("Altura de 1 a 8: ")
    imprimir(tamanho)



def imprimir(tamanho):

    for i in range(tamanho):
        i = i+1
        j = (tamanho-i)
        print(" " *j, end="#" *i)
        j =- 1
        print("  ", end="")
        print("#" *i, end="")
        print()


main()